packages/vscode/src/shared/rpc.ts

```diff
@@ -0,0 +1,108 @@
+import type { ChildProcess } from 'node:child_process';
+
+export function expose(
```

Comment:
i think it's better to use https://github.com/antfu-collective/birpc instead of maintaining the RPC methods.

Reply:
birpc is also being used in `@rstest/core` now. 😃